### PR TITLE
Debug add child

### PR DESCRIPTION
**Pull request by mmazanec22**
Deal with errors that were preventing add child and messing with error focus

### CODE REVIEW

**Review by kmagida110 — APPROVED** [review]
Looks good!

File: client/src/containers/AddChild/AddChild.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ const AddChild: React.FC = () => {
     })
       .then((updatedChild) => {
         updateChild(updatedChild);
-        if (!updatedChild || !steps || !indexOfCurrentStep) return;
+        if (!updatedChild) return;
```
Comment (bamader) [inline client/src/containers/AddChild/AddChild.tsx:105]: Nit: Should we do these conditional checks before you invoke `updateChild` to modify the state? That way we don't replace anything if the server doesn't send back a correctly updated child?

**Review by bamader — APPROVED** [review]
LGTM. Just one question on the ordering of a null check vs updating state.

File: client/src/containers/AddChild/AddChild.tsx (hunk 1 above)
Comment (jlhogan) [reply client/src/containers/AddChild/AddChild.tsx:105]: we can expect that updatedChild will always have a value if we're in the `then` block

**Review by jlhogan — APPROVED** [review]

**Pull request merged**